app/src/main/java/org/oppia/android/app/home/HomeFragmentPresenter.kt

```diff
@@ -41,7 +41,8 @@ class HomeFragmentPresenter @Inject constructor(
   @TopicHtmlParserEntityType private val topicEntityType: String,
   @StoryHtmlParserEntityType private val storyEntityType: String,
   private val resourceHandler: AppLanguageResourceHandler,
-  private val dateTimeUtil: DateTimeUtil
+  private val dateTimeUtil: DateTimeUtil,
+  private val multiTypeAdapterFactory: BindableAdapter.MultiTypeBuilder.Factory
```

Comment:
For consistency: `multiTypeBuilderFactory`

Reply:
Noted.

Reply:
This appears addressed in the latest code. Marking as resolved since Rajat has already approved the PR.